integration-test/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
   master2603:
     build:
       context: ..
-      dockerfile: integration-test/Dockerfile.jenkins.2603
```

Comment:
What's the logic on dropping the specific version. I would have presumed that we locked this to .2603 and .2403 so we can confirm support on particular versions. 

If we are choosing to drop this for just lts. Can we remove either master2604 or master2463 as it seems redundant. Unless I'm missing something.

Reply:
Fixed